server/services/licenceService.ts

```diff
@@ -198,6 +201,39 @@ export default class LicenceService {
     return this.licenceApiClient.updateAdditionalConditionData(licenceId, additionalConditionId, requestBody, user)
   }
 
+  async uploadExclusionZoneFile(
+    licenceId: string,
+    additionalConditionId: string,
+    fileToUpload: Express.Multer.File,
+    user: User,
+    testMode = false
+  ): Promise<void> {
+    logger.info(`Called uploadExclusionZoneFile - name ${fileToUpload.originalname} path ${fileToUpload.path}`)
+    await this.licenceApiClient.uploadExclusionZoneFile(licenceId, additionalConditionId, user, fileToUpload)
+    if (!testMode) {
+      logger.info(`Removing file ${fileToUpload.path} after uploading content to API`)
+      fs.unlinkSync(fileToUpload.path)
+    }
+  }
+
+  async removeExclusionZoneFile(licenceId: string, conditionId: string, user: User): Promise<void> {
+    logger.info(`removeExclusionZoneFile - licenceId ${licenceId}, conditionId  ${conditionId}`)
+    return this.licenceApiClient.removeExclusionZoneFile(licenceId, conditionId, user)
+  }
+
+  // Get the streamed image data for rendering in HTML templates
+  async getExclusionZoneImage(licenceId: string, conditionId: string, user: User): Promise<Readable> {
```

Comment:
Unit test?

Reply:
it('Get the exclusion zone map image as base64 JPEG', async () ? This one? Ah, you mean the other implementation which streams the data? Its not actually used anywhere as yet. I can add a test for it.

Reply:
Added a test for this too, even though unused currently.